src/Endpoints/Payments.php

```diff
@@ -335,6 +337,37 @@ public function addOrder($id, $orderData, $overwrite = false)
         return new Order(end($res->json));
     }
 
+    /**
+     * Add order status to Alma Order by merchant_order_reference
+     *
+     * @param string $paymentId
+     * @param string $merchantOrderReference
+     * @param string $status
+     * @param bool | null $isShipped
+     * @return void
+     * @throws ParametersException
+     * @throws RequestError
+     * @throws RequestException
+     */
+    public function addOrderStatusByMerchantOrderReference(
+        $paymentId,
+        $merchantOrderReference,
+        $status,
+        $isShipped = null)
+    {
+        $this->checkAddOrderStatusParams($paymentId, $merchantOrderReference, $status, $isShipped);
```

Comment:
Not fan to check 4 params in only one call:
-> one call per param
or
-> test here